files/zh-cn/learn/css/building_blocks/the_box_model/index.md

```diff
@@ -219,7 +219,7 @@ html {
 
 ### 边框
 
-边框是在边距和填充框之间绘制的。如果你正在使用标准的盒模型，边框的大小将添加到框的宽度和高度。如果你使用的是替代盒模型，那么边框的大小会使内容框更小，因为它会占用一些可用的宽度和高度。
+边框是在边距和填充框之间绘制的。如果你正在使用标准的盒模型，边框的大小将添加到框的宽度和高度。如果你使用的是替代盒模型，边框越大会使内容框越小，因为它会占用一些可用的宽度和高度。
```

Comment:
In en-US, we have the following description:

> If you are using the alternative box model then the size of the border makes the content box smaller as it takes up some of that available width and height of the element box.

So, could you translate this as following:

```suggestion
边框是在边距和填充框之间绘制的。如果你正在使用标准的盒模型，边框的大小将添加到框的宽度和高度。如果你使用的是替代盒模型，边框会使内容框缩小，因为它会占用一些可用的宽度和高度。
```

Reply:
I understant, but the English version of the statement is also inaccurate. Please wait, I will fixed en-US version too.